numba/tests/test_runtests.py

```diff
@@ -116,6 +116,23 @@ def filter_test(xs):
         # The tests must be equivalent
         self.assertEqual(sorted(full), sorted(sliced))
 
+    def test_gitdiff(self):
+        # Check for git
+        try:
+            subprocess.call("git", stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+        except FileNotFoundError as e:
+            self.skipTest("no git available")
+
+        # default
+        outs = self.get_testsuite_listing(['-g'])
+        self.assertNotIn("Git diff by common ancestor", outs)
+        # using ancestor
+        outs = self.get_testsuite_listing(['-g=ancestor'])
+        self.assertIn("Git diff by common ancestor", outs)
+        # misspelled ancestor
+        with self.assertRaises(subprocess.CalledProcessError):
+            self.get_testsuite_listing(['-g=ancest'])
```

Comment:
This adds noise to the test output as the error streams aren't captured/redirected?

Reply:
ok, i'll sink it to devnull